js/index.js

```diff
@@ -1,7 +1,63 @@
-// Iteration 1: Names and Input
+console.log("I'm ready")
 
+// Iteration 1: Names and Input
+const hacker1 = "Marie";
+console.log(`"The driver's name is ${hacker1}"`);
+const hacker2 = "Yosra";
+console.log(`"The navigator's name is ${hacker2}"`);
 
 // Iteration 2: Conditionals
+if(hacker1.length > hacker2.length) {
+    console.log(`The driver has the longest name, it has ${hacker1.length} characters.`)
+} else if(hacker1.length < hacker2.length) {
+    console.log(`It seems that the navigator has the longest name, it has ${hacker2.length} characters.`)
+} else {
+    console.log(`Wow, you both have equally long names, ${hacker1} characters!`)
+}
+
+// Iteration 3: Loops
+let newDriverName = "";
+for (let i = 0; i < hacker1.length; i++) {
+    newDriverName += hacker1[i].toUpperCase() + " "
+}
+console.log(newDriverName);
+
+let newNavigatorName = "";
+for (let i = hacker2.length-1; i >= 0; i--) {
+    newNavigatorName += hacker2[i]
+}
+console.log(newNavigatorName);
+
+if (hacker1.localeCompare(hacker2) === -1){
+    console.log(`The driver's name goes first.`)
+} else if ((hacker2.localeCompare(hacker1) === 1)){
```

Comment:
hacker1.localeCompare(hacker2) written above returns -1 if hacker1 goes first
hacker2.localeCompare(hacker1)  written here returns 1 if hacker1 goes first.

Since you changed your localeCompare AND what it returns you basically have the same condition on both of these conditions, meaning hacker2 will never go first:)